package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/client": "^3.3.15",
```

Comment:
Important to note for everyone: We need to install apollo client and remove other apollo related packages.

Reply:
Thanks for the heads up!

Reply:
you mean run `npm install` when we pull his down?